docs/src/docs/asciidoc/user/expiry.adoc

```diff
@@ -84,3 +84,6 @@ include::{sourcedir32}/xml/src/test/resources/configs/docs/getting-started.xml[t
 ----
 
 <1> Simply pass the fully qualified class name of your custom expiry.
+
+
+Ehcache2 to Ehcache3 Expiry migration: <<migration-guide.adoc#custom-expiry,_Custom Expiry_>>
```

Comment:
The linking from here is good but I would preface this with a bit more text explaining it and link in text instead of having text then link.